content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -15,12 +15,24 @@ export class _StartupOverlay extends React.PureComponent {
 
     this.state = {
       emailInput: "",
-      overlayRemoved: false
+      overlayRemoved: false,
+      flowId: "",
+      flowBeginTime: ""
     };
     this.initScene();
   }
 
-  initScene() {
+  async initScene() {
```

Comment:
Call this in `componentDidMount`, I just noticed this was in the constructor and it shouldn't be there